### PR TITLE
Prevent abductors from abducting the disk

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
## About The Pull Request

Pretty self-explanatory.

## Why It's Good For The Game

Yeah.

## Changelog
:cl: Naksu
fix: the nuclear disk is now abductor-proof.
/:cl:

### CODE REVIEW

File: code/datums/components/stationloving.dm [hunk 1]
```diff
@@ -56,13 +56,16 @@
 
 /datum/component/stationloving/proc/in_bounds()
 	var/static/list/allowed_shuttles = typecacheof(list(/area/shuttle/syndicate, /area/shuttle/escape, /area/shuttle/pod_1, /area/shuttle/pod_2, /area/shuttle/pod_3, /area/shuttle/pod_4))
+	var/static/list/disallowed_centcom_areas = typecacheof(list(/area/abductor_ship, /area/fabric_of_reality, /area/awaymission/errorroom))
 	var/turf/T = get_turf(parent)
 	if (!T)
 		return FALSE
 	var/area/A = T.loc
-	if(istype(A, /area/fabric_of_reality)) // Obviously terrible, just for test merging
```
Comment (JJRcop) [inline code/datums/components/stationloving.dm]: for_testing_do_not_publish_to_live_env_i_mean_it_looking_at_you_tim

**Comment by Shadowflame909** [comment]
Uh. Correct me if I'm wrong. But doesn't the disk have an actual purpose to go off onto the Centcomm Z-Level?

Specifically Nuke Ops. It gives you a minor loss if it stays on the station, and a station wins if it's on the shuttle.

It also causes a Catastrophic Failure when you go to the Nuke Ops base and nuke their own base.

Nuke Ops Proof this p l e a s e

**Comment by Arianya** [comment]
>Edit: I might just be an idiot who doesn't understand the code.

Correct

**Comment by JJRcop** [comment]
@Shadowflame909 This PR checks the disk's area against a blacklist if it's on the centcom zlevel. If it's not in the blacklist nothing happens.

File: code/datums/components/stationloving.dm (hunk 1 above)
Comment (PKPenguin321) [reply code/datums/components/stationloving.dm]: this has been in the code since May 25 of 2018, so a year + about a week
incredible

**Review by 81Denton — APPROVED** [review]

**Pull request merged**